external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaMicroBatchSourceSuite.scala

```diff
@@ -199,6 +199,41 @@ abstract class KafkaMicroBatchSourceSuiteBase extends KafkaSourceSuiteBase {
       StopStream)
   }
 
+
+  test("Rate limit set to Long.Max should not overflow integer during end offset calculation[SPARK-26718]") {
```

Comment:
nit: There's actually no rule on this, but most of the cases JIRA title is used as a prefix (search "SPARK-" against `*Suite.scala`) so might be better to follow the case.